starters/features/leaflet-map/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "description": "Use Leaflet Maps in your Qwik app",
+    "__qwik__": {
+      "displayName": "Integration: Leaflet Maps",
+      "priority": -10,
+      "viteConfig": {},
+      "docs": [
+        "https://leafletjs.com/reference.html",
+        "https://leafletjs.com/examples/quick-start/",
+        "https://www.youtube.com/watch?v=_rQH9EwUwn4&list=PLaaTcPGicjqgLAUhR_grKBGCXbyKaP7qR"
```

Comment:
My only concern is that this video in not in english and it's useless for those who don't speak Spanish.the two official doc links are enough to me.

Reply:
I add this extra references (all in English) official and very interesting with many examples and explains:
- https://tomik23.github.io/leaflet-examples/
- https://www.sitepoint.com/leaflet-create-map-beginner-guide/

Reply:
Thanks